examples/interpretation/notebooks/Dutch_F3_patch_model_training_and_evaluation.ipynb

```diff
@@ -747,28 +726,52 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# add model checkpointing\n",
-    "checkpoint_handler = ModelCheckpoint(\n",
-    "    output_dir,\n",
-    "    \"model_f3_nb\",\n",
-    "    save_interval=1,\n",
-    "    n_saved=1,\n",
-    "    create_dir=True,\n",
-    "    require_empty=False,\n",
+    "transform_fn = lambda output_dict: (output_dict[\"y_pred\"].squeeze(), output_dict[\"mask\"].squeeze())\n",
+    "evaluator = create_supervised_evaluator(\n",
+    "    model,\n",
+    "    prepare_batch,\n",
+    "    metrics={\n",
+    "        \"nll\": Loss(criterion, output_transform=transform_fn),\n",
+    "        \"pixacc\": pixelwise_accuracy(n_classes, output_transform=transform_fn, device=device),\n",
+    "        \"cacc\": class_accuracy(n_classes, output_transform=transform_fn),\n",
+    "        \"mca\": mean_class_accuracy(n_classes, output_transform=transform_fn),\n",
+    "        \"ciou\": class_iou(n_classes, output_transform=transform_fn),\n",
+    "        \"mIoU\": mean_iou(n_classes, output_transform=transform_fn),\n",
+    "    },\n",
+    "    device=device,\n",
     ")\n",
+    "trainer.add_event_handler(Events.ITERATION_STARTED, scheduler)\n",
     "\n",
+    "# Logging:\n",
     "trainer.add_event_handler(\n",
-    "    Events.EPOCH_COMPLETED, checkpoint_handler, {config.MODEL.NAME: model}\n",
-    ")"
+    "    Events.ITERATION_COMPLETED, logging_handlers.log_training_output(log_interval=config.PRINT_FREQ),\n",
+    ")\n",
+    "trainer.add_event_handler(Events.EPOCH_COMPLETED, logging_handlers.log_lr(optimizer))\n",
+    "\n",
+    "# Tensorboard and Logging:\n",
+    "trainer.add_event_handler(Events.ITERATION_COMPLETED, tensorboard_handlers.log_training_output(summary_writer))\n",
+    "trainer.add_event_handler(Events.ITERATION_COMPLETED, tensorboard_handlers.log_validation_output(summary_writer))\n",
+    "\n",
+    "# add specific logger which also triggers printed metrics on test set\n",
+    "@trainer.on(Events.EPOCH_COMPLETED)\n",
+    "def log_training_results(engine):\n",
+    "    evaluator.run(train_loader)\n",
+    "    log_results(engine, evaluator, summary_writer, n_classes, stage=\"Training\")\n",
```

Comment:
Can we have these two functions follow the same import pattern? Like either use:
```
tensorboard_handlers.log_results( ..
logging_handlers.log_metrics( ..
```
or 
```
log_results( ..
log_metrics( ..
```
i think it looks cleaner that way

Reply:
I always like specifying full library path - will go with the former